### DISCUSSION TITLE
Backup and restore settings

### DISCUSSION THREAD

**Discussion by alanlivio** [microsoft/terminal#16368] [Q&A]
Hi 👋
The PowerToys tools have a feature to backup and restore settings. See the image bellow.
Is there any similar feature for WindowsTerminal?

![Screenshot 2023-11-24 091300](https://github.com/microsoft/terminal/assets/604808/9184c9a0-9b76-4245-9c0b-c43f306fd4ab)

**Comment by lhecker** [comment]
Not yet! And I couldn't find any feature request for it either. 🤔
